### DISCUSSION TITLE
p50 GitHubの誤植

### DISCUSSION THREAD

**Discussion by hajipy** [budougumi0617/go_todo_app#89] [Ideas]
> たとえば、あるGitHubeにあるリポジトリが1モジュールで、該当リポジトリの各ディレクトリがパッケージになります。

GitHubeの最後のeが不要かと思います。

**Comment by budougumi0617** [comment]
こちらもご指摘のとおりでした。エラッタに追記させていただきました。ご報告ありがとうございました！！

https://github.com/budougumi0617/go_todo_app/releases/tag/v1.1.1
